modules/ROOT/pages/afm-download-upload-monitor.adoc

```diff
@@ -0,0 +1,119 @@
+* Figure out the frequency at which you would like the monitor to test the endpoints.
++
+The shortest interval is every fifteen minutes. You can also choose to run the monitor hourly, daily, weekly, or monthly. The schedule begins immediately after you create the monitor. You can schedule another monitor later for the same test suite that is more detailed, if you prefer. You can also modify the schedule after you create the monitor, if you want to specify any of these options:
+** Set an hourly schedule to start at a certain number of minutes after the hour begins.
+** Set a daily, weekly, or monthly schedule to start at a certain time of day.
+* Decide whether you would like the request messages to include additional header information.
+* Obtain one of the following pieces of information, if you would like to send results of failed tests to one of these reporting tools:
++
+|===
+|Reporting Tool |Required Information
+
+|NewRelic
+|License Key
+
+|PagerDuty
+|RoutingKey
+
+|Slack
+|Webhook
+
+|SumoLogic
+|Endpoint
+
+|Email
+|Email address
+|===
+
+
+== Procedures
+
+To download a copy of a monitor:
+
+. In the list of your monitors, find the monitor that you want to download a copy of and select it.
+. Click the three dots next to the name of the monitor and select *Download Monitor*.
++
+*Result:* A ZIP file is downloaded to your computer. The filename is `<your Anypoint Platform group ID>_<the name of the monitor>.zip`.
+
+To create a monitor by uploading a test suite:
+
+. At the top of the page that lists your monitors, click *Upload Monitor*.
+. Upload the folder that contains your test suite.
+. In the *Monitor Setup* section, specify the name of the monitor, the location from which to run the test, how often to run the test, and whether to validate SSL certificates.
+.. Specify a name for the monitor.
+.. Select the location from which to run the test.
++
+Public locations are grouped separately from private locations in the list. You can create a private location by clicking an option in the list.
+.. Specify the intervals at which you want the monitor to run the test.
++
+The interval that you select begins when you click *Create Monitor* at the end of this procedure.
+.. Toggle the *SSL Certificate Validation* switch off if the endpoints that your monitor will test use self-signed certificates.
++
+If they do use self-signed certificates and you do not toggle this switch off, tests against those endpoints fail.
++
+** If the toggle is on, all endpoints that use SSL certificates must have certificates that are signed by a trusted authority.
+** If the toggle is off, all endpoints that use SSL certificates must be self-signed.
++
+To test a combination of endpoints that use trusted SSL certificates and endpoints that use self-signed certificates, create two separate monitors: one for testing the first group of endpoints, the other for testing the second group of endpoints.
```

Comment:
See previous edit suggestions for this step, which is pretty redundant, long, and complicated.